migration-engine/connectors/sql-migration-connector/src/sql_schema_differ.rs

```diff
@@ -21,15 +21,29 @@ use sql_schema_describer::walkers::{EnumWalker, ForeignKeyWalker, TableWalker};
 use std::collections::HashSet;
 use table::TableDiffer;
 
+pub(crate) fn diff_schemas(
```

Comment:
maybe rename to `calculate`, and you can call it `sql_schema_differ::calculate`.

I'm a huge proponent on creating pure functions and calling them using the qualified module.

Reply:
I like this convention, let's do that.